src/content/en/fundamentals/performance/rail.md

```diff
@@ -132,21 +142,16 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps on an icon or button (for example, opening the nav menu, tapping Compose).</td>
+      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
-      <td data-th="User Test">User drags their finger and the app's response is bound to the finger position (for example, pull to refresh, swiping a carousel).</td>
-    </tr>
```

Comment:
I'd be fine with this as an animation, as long as folks won't get confused between the start of a touch drag, and the continuous phase of a touch drag.

The continuous phase of a touch drag is definitely animation.